todo_core/settings.py

```diff
@@ -0,0 +1,132 @@
+"""
+Django settings for todo_core project.
+
+Generated by 'django-admin startproject' using Django 4.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+import os.path
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = (
+    "django-insecure-m+splp6qzy016*bt6s8(2i_2s=$#q_tosty9&lk%5h%!dhjikp"

```

Comment:
Do not place your secret key in a public repository, take it from the environment variable or from a file not included in the repo.